src/main/scala/cromwell/util/GcsUtil.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.util
+
+object GcsUtil {
```

Comment:
It'd be good to never need a Util object in our codebase. This could be added as a method on `GoogleCloudStoragePath` via a construct called an `implicit value class`, it'd look like this (I'm typing this freehand, might not be perfect):

    implicit class EnhancedGoogleCloudStoragePath(val p: GoogleCloudStoragePath) extends AnyVal {
      def slurp(clientSecrets: String): String = new String(........)
  }

If you import that where you want it and if `foo` is a `GoogleCloudStoragePath` you can just call `foo.slurp(mySecrets)`.